src/Keboola/DbExtractor/Application.php

```diff
@@ -68,12 +76,12 @@ public function run(): void
             throw new UserException(sprintf("Action '%s' does not exist.", $this->getConfig()->getAction()));
         }
 
-        $this->$actionMethod();
-    }
+        $result = $this->$actionMethod();
 
-    public function setConfigDefinition(ConfigurationInterface $definition): void
-    {
-        $this->configDefinition = $definition;
+        print json_encode(
```

Comment:
Nevypíše to `null`, pokud ta action metoda nic nevrátí?

Reply:
Jo, jasně - ono to vrátí nějaký ten state s něčím. To ale teda nevím, jestli by se nutně mělo vypisovat.

Reply:
Akce se bud ukonci nekterou z exceptions nebo se provede spravne a vypise result - do ted se vypisoval, tak jsem to zachoval. Pro `run` je to asi zbytecne, ale pro ty synchronni akce je to hodne zasadni.

Reply:
Jo, sorry, nevšiml jsem si, že to předtím vypisovalo vždycky.